### PR TITLE
Bump npm dependencies

### PR DESCRIPTION
**Pull request by RensR**

### CODE REVIEW

File: .github/actions/split-tests/.npmrc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+auto-install-peers=true
+strict-peer-dependencies=false
```
Comment (RensR) [inline .github/actions/split-tests/.npmrc:2]: required due to lockfileVersion 6.0

File: contracts/hardhat.config.ts [hunk 2]
```diff
@@ -54,6 +54,7 @@ let config = {
       allowUnlimitedContractSize: Boolean(
         process.env.ALLOW_UNLIMITED_CONTRACT_SIZE,
       ),
+      hardfork: 'merge',
```
Comment (RensR) [inline contracts/hardhat.config.ts:57]: required due to upgrade to hardhat 2.17 which defaults to shanghai which causes a test to fail

File: contracts/test/v0.8/automation/UpkeepTranscoder.test.ts [hunk 3]
```diff
@@ -31,10 +31,7 @@ describe('UpkeepTranscoder', () => {
     const encodedData = '0xc0ffee'
 
     it('reverts if the from type is not an enum value', async () => {
-      await evmRevert(
```
Comment (RensR) [inline contracts/test/v0.8/automation/UpkeepTranscoder.test.ts:34]: @smartcontractkit/keepers this test failed after bumping Hardhat as it didn't revert with the given message. It still reverts so I removed the message component

Comment (RyanRHall) [reply contracts/test/v0.8/automation/UpkeepTranscoder.test.ts:34]: I'm not worried about it

**Review by RyanRHall — APPROVED** [review]

**Pull request merged**